index.html

```diff
@@ -50,5 +50,43 @@
         </div>
     </div>
 </header>
+<div class="mobile__menu">
+    <button class="mobile__menu__close__button">
+        <img class="" src="/img/x.svg">
+    </button>
+    <ul  class="mobile__menu__list">
+        <li class="mobile__menu__list__item"><a  href="#">Главная</a></li><br>
+        <li class="mobile__menu__list__item"><a  href="#">Цены</a></li><br>
+        <li class="mobile__menu__list__item"><a  href="#">Контакты</a></li><br>
+        <li class="mobile__menu__list__item"><a  href="#">Статьи</a></li><br>
+        <li class="mobile__menu__list__item"><a  href="#">Юр.лицам</a></li><br>
+        <li class="mobile__menu__list__item"><a href="#">Услуги</a></li><br>
+    </ul>
+    <div class="mobile__menu__contact">
+        <div class="mobile__menu__contact__item">
+            <a href="tel:+7(919)777-90-09">+7 (919) 777-90-09</a><br>
+            <span class="mobile__menu__contact__span">24/7 Круглосуточно</span>
+        </div>
+        <a class="mobile__menu__contact__icon" href="https://wa.me/+7(919)777-90-09">
+            <img src="img/whatsapp.svg" alt="WhatsApp">
+            <span>Чат WhatsApp</span>
+        </a><br>

```

Comment:
`<br>` мы не используем потому что он не стилизуется под респонсив
```suggestion
        </a>

```